topo/topo_test.go

```diff
@@ -184,6 +184,10 @@ func (f *defaultFakeTopology) ConfigPush(context.Context, string, io.Reader) err
 	return nil
```

Comment:
why no test?

Reply:
yeah looks like this file in general needs more coverage for all of the topo manager methods

I added a task to increase the coverage for this file in general